Improve Service Catalog Overview document for custom resources (#5975)
kyma-project/kyma: docs/service-catalog/03-01-resources.md

* **Secret** is a basic resource to transfer credentials or configuration details that the application uses to consume a ServiceInstance. The service binding process leads to the creation of a Secret.

These are the Service Catalog resources that Kyma provides:

> * [**ServiceBindingUsage**](#custom-resource-service-binding-usage) is a Kyma custom resource that allows the ServiceBindingUsage controller to inject Secrets into a given application.

Reviewer: ```suggestion
* [**ServiceBindingUsage**](#custom-resource-service-binding-usage) - a Kyma custom resource that allows the ServiceBindingUsage controller to inject Secrets into a given application.
```
I'd change the `is`'s to `-`'s, as you already introduce the resources in line 30 and then list them :)

> * [**UsageKind**](#custom-resource-usage-kind) is a Kyma custom resource that defines which resources can be bound with the ServiceBinding and how to bind them.

Reviewer: ```suggestion
* [**UsageKind**](#custom-resource-usage-kind) - a Kyma custom resource that defines which resources you can bind to the ServiceBinding and how to bind them.
```
